wren-modeling-rs/core/src/mdl/context.rs

```diff
@@ -24,8 +25,10 @@ pub async fn create_ctx_with_mdl(
     ctx: &SessionContext,
     analyzed_mdl: Arc<AnalyzedWrenMDL>,
 ) -> Result<SessionContext> {
-    let new_state = ctx
-        .state()
+    let config = ctx
+        .copied_config()
+        .with_create_default_catalog_and_schema(false);
+    let new_state = SessionStateBuilder::new_from_existing(ctx.state())
```

Comment:
They should be removed after https://github.com/apache/datafusion/issues/11988 fixed